rakefile.rb

```diff
@@ -17,14 +17,16 @@
 
 	#Ignore files from automatic harvesting that will be installed specifically
 	harvest_ignored_files = [
-		'MoBi.exe' 
+		'MoBi.exe',
+		'Standard Molecule.pkml'
 	]
 
 	#Files required for setup creation only
 	setup_files	 = [
 		'packages/**/OSPSuite.Core/**/*.xml',
 		'packages/**/OSPSuite.Presentation/**/*.{wxs,xml}',
 		'packages/**/OSPSuite.TeXReporting/**/*.*',
+		'examples/**/*.{wxs,pkml,mbp3}',
```

Comment:
I need to add those guys specifically (I do not use *.* otherwise we might get all other stuff from the submodule)